pkg/component/component.go

```diff
@@ -502,13 +498,6 @@ func ValidateComponentCreateRequest(client *occlient.Client, componentSettings c
 		return errors.Wrapf(err, "failed to check component of name %s", *componentSettings.Name)
 	}
 
-	// If component does not exist, create it
-	if isCmpExistsCheck {
-		if isCmpExists {
```

Comment:
the function was always called with `isCmpExistsCheck = false`, so this was never executed, probably some relict from before